components/molecules/charts/schedule/helper.js

```diff
@@ -0,0 +1,292 @@
+import React from 'react'
```

Comment:
@Ojaghi1991 Seperate into indivudal files - this helps with code bloat if we dont need to use all the methods in here - easier for testing and code maintenance also

camelCase we have a util for already - dont need lodash for that i thhink

Also prefer to remove lodash if we can from all files